packages/cbioportal-utils/src/index.tsx

```diff
@@ -19,6 +19,8 @@ export * from './model/MyCancerGenome';
 export * from './model/MyVariantInfo';
 export * from './model/OncoKB';
 export * from './model/RemoteData';
+export * from './model/signal/SignalMutation';
+export * from './model/signal/SignalMutationStatus';
```

Comment:
I think it is okay to put these directly under model:

```suggestion
export * from './model/SignalMutation';
export * from './model/SignalMutationStatus';
```